pkg/metricscollector/v1beta1/file-metricscollector/file-metricscollector.go

```diff
@@ -34,15 +35,27 @@ import (
 	"github.com/kubeflow/katib/pkg/metricscollector/v1beta1/common"
 )
 
+var (
+	ErrFileFormat = fmt.Errorf("format must be set %v or %v", commonv1beta1.TextFormat, commonv1beta1.JsonFormat)
+	ErrOpenFile   = errors.New("failed to open the file")
+	ErrReadFile   = errors.New("failed to read the file")
+	ErrParseJson  = errors.New("failed to parse the json object")
```

Comment:
```suggestion
	errFileFormat = fmt.Errorf("format must be set %v or %v", commonv1beta1.TextFormat, commonv1beta1.JsonFormat)
	errOpenFile   = errors.New("failed to open the file")
	errReadFile   = errors.New("failed to read the file")
	errParseJson  = errors.New("failed to parse the json object")
```

No need to expose these vars.

Reply:
okay, I'll change it.